src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -315,8 +231,6 @@ def __init__(self, config, layer_number=None):
         self.num_heads = config.n_head
         self.head_dim = self.hidden_size // self.num_heads
         self.split_size = self.hidden_size
-        self.attention_softmax_in_fp32 = config.attention_softmax_in_fp32
```

Comment:
This is the only place where this config parameter is used so it should also be removed from the config?

Reply:
yes, I have removed it in fa40cc81ecc6674928cd051c4720f5f2e3808d3a